### ISSUE TITLE
Move all strings to a yaml file `.yml` for translation

### ISSUE THREAD

**Issue by t1-tracey** [sky-music/sky-python-music-sheet-maker#110]
this is so that different languages can be added (thatskybot is intended to have Japanese strings as well). this is a lower priority issue; I'm just writing it down so I remember for later on.

**Comment by jmmelko** [comment]
After #120 has been addressed

**Comment by jdewfiez** [comment]
Still waiting for this feature

**Comment by jmmelko** [comment]
Do you have any python skills?
If yes, please fee free to do it.
I have been busy setting up the music cog in the bot. And it works now!

Usually it is Tracey who takes care of yaml files. I will try to do it on my spare time.

**Comment by t1-tracey** [comment]
Wow this is nice work. @jdewfiez, JMM set up the languages here, feel free to duplicate and edit the text for your own language. (be sure to be on the dev branch)

https://github.com/sky-music/sky-python-music-sheet-maker/tree/dev/python/langs

**Comment by jdewfiez** [comment]
I'll start translating it to Vietnamese now.
_I was learning Japanese but got lazy instead :(_

**Comment by t1-tracey** [comment]
I only know greetings and basic phrases in Japanese, I’ll see if I can find player(s) willing to translate.
I think we can close this issue for now, later on we can use a tool like [Crowdin](https://crowdin.com/) or [Transifex](https://www.transifex.com/) for translations (they are free for open source projects)

**Comment by jmmelko** [comment]
It is urgent to review the questions in English first (I know you have Tracey, especially regarding the wal of text), because if we start translating and then change all the questions, the translators will have to work again.
